Makefile

```diff
@@ -13,9 +13,13 @@ build:
 	cd cmd/state-handler && go fmt && go vet && go build
 	cd cmd/policy-handler && go fmt && go vet && go build
 
-docker:
+docker-state-handler:
 	docker build -f cmd/state-handler/Dockerfile -t $(IMAGE_REGISTRY)/$(STATE_HANDLER_IMAGE):$(IMAGE_TAG) .
-	docker build -f cmd/policy-handler/Dockerfile -t $(IMAGE_REGISTRY)/$(POLICY_HANDLER_IMAGE):$(IMAGE_TAG) .
+
+docker-policy-handler:
```

Comment:
you also need to split the docker-push command.

the CI is failing because of that.

Reply:
@SchSeba well the failure is using the STATE image at policy hander, let me fix it.